configure.ac

```diff
@@ -35,6 +35,50 @@ PKG_CHECK_MODULES(SDL, [sdl2 >= 2.0.1])
 PKG_CHECK_MODULES(SDLMIXER, [SDL2_mixer >= 2.0.0])
 PKG_CHECK_MODULES(SDLNET, [SDL2_net >= 2.0.0])
 
+# Check for bash-completion.
```

Comment:
These are all enabled by default, right?

Reply:
Yes, the default behavior here is to have all of these options enabled; unless Python is not found, in which case the script warns the user and automatically switches to disabled.

Reply:
Hmm, I suppose the "check" wording in the comments could be confusing. Would you like me to change that?

Reply:
Honestly, I don't mind.